internals/cli/cli.go

```diff
@@ -49,70 +49,59 @@ var (
 // the pebble client.
 const defaultPebbleDir = "/var/lib/pebble/default"
 
-type options struct {
-	Version func() `long:"version"`
+// ArgumentHelp contains help information about the positional arguments accepted
+// by a command.
+type ArgumentHelp struct {
```

Comment:
This looks unused. It'd be good to perform a full self-review to make sure there are no left overs from previous iterations.